examples/basic/file_operations_client.py

```diff
@@ -94,7 +92,9 @@ def run(api: DataTransferApi, local_path: str, remote_path: Optional[str] = None
     op = api.wait_for([op.id])
     log.info(f"Operation {op[0].state}")
 
-
+####################################
+# Main function
+# =============
 def main(
```

Comment:
Is the "help" information coming from elsewhere? If this is the actual source for the help documentation, we should add concluding punctuation.